Jenkinsfile

```diff
@@ -464,7 +464,7 @@ pipeline {
         }
         when {
           expression {
-            skipStageType(failedStageTypes, env.PROFILE) && (withLabels(getLabels(env.PROFILE)) || withDbLabels(env.DB))
+            skipStageType(failedStageTypes, env.PROFILE) && (withLabels(cambpmGetLabels(env.PROFILE)) || withDbLabels(env.DB))
```

Comment:
```suggestion
            skipStageType(failedStageTypes, env.PROFILE) && withLabels(cambpmGetLabels(env.PROFILE).minus('cockroachdb'))
```

Reply:
I adjusted the `cambpmGetLabels` method, so that it now accepts an optional `String...` argument, which we can use to remove the redundant labels.